.docker/ESPhomeBeta/compose.yml

```
@@ -0,0 +1,14 @@
+  esphome:
+    container_name: esphomebeta
+    image: ghcr.io/esphome/esphome:latest
+    volumes:
+      - /path/to/esphome/config:/config
+      - /etc/localtime:/etc/localtime:ro
+    restart: always
+    privileged: true
+    network_mode: host
+    environment:
+      - USERNAME=test
+      - PASSWORD=ChangeMe
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)